AEGIS.IO/GeoJson/GeoJsonReader.cs

```diff
@@ -0,0 +1,536 @@
+                    case "Polygon":
+                        if (obj.Coordinates == null)
+                            throw new IOException("Geometry coordinates value is null.");
+                        return CreatePolygon(obj.Coordinates, properties);
+                    case "MultiPolygon":
+                        if (obj.Coordinates == null)
+                            throw new IOException("Geometry coordinates value is null.");
+                        return CreateMultiPolygon(obj.Coordinates, properties);
+                    case "GeometryCollection":
+                        if (obj.Geometries == null)
+                            throw new IOException("GeometryCollection geometries value is null.");
+                        return CreateGeometryCollection(obj.Geometries, properties);
+                    default: throw new ArgumentException("Not supported geometry type: " + obj.Type + ".");
+                }
+            }
+            catch (Exception ex)
+            {
+                throw new IOException(MessageContentInvalid, ex);
+            }
+        }
+
+        #endregion
+
+        #region Private methods
+        /// <summary>
+        /// Determines the ReferenceSystem from the Crs object.
+        /// </summary>
+        /// <param name="obj">The input crs object.</param>
+        /// <returns>The determined ReferenceSystem.</returns>
+        private IReferenceSystem GetReferenceSystem(CrsObject obj)
+        {
+            if (obj == null)
+                return GeographicCoordinateReferenceSystems.FromName("WGS84").FirstOrDefault();
+
+            if (obj.Type == "name")
+            {
+                if (obj.Properties.ContainsKey("name"))
+                {
+                    IList<IReferenceSystem> systems = GeocentricCoordinateReferenceSystems.FromIdentifier(obj.Properties["name"]).ToList<IReferenceSystem>();
+                    if (systems.Count > 0)
+                        return systems[0];
+
+                    systems = GeographicCoordinateReferenceSystems.FromIdentifier(obj.Properties["name"]).ToList<IReferenceSystem>();
+                    if (systems.Count > 0)
+                        return systems[0];
+
+                    systems = GridReferenceSystems.FromIdentifier(obj.Properties["name"]).ToList<IReferenceSystem>();
+                    if (systems.Count > 0)
+                        return systems[0];
+
+                    systems = ProjectedCoordinateReferenceSystems.FromIdentifier(obj.Properties["name"]).ToList<IReferenceSystem>();
+                    if (systems.Count > 0)
+                        return systems[0];
+
+                    systems = VerticalCoordinateReferenceSystems.FromIdentifier(obj.Properties["name"]).ToList<IReferenceSystem>();
+                    if (systems.Count > 0)
+                        return systems[0];
+
+                    return GeographicCoordinateReferenceSystems.FromName("WGS84").FirstOrDefault();
```

Comment:
Now throwing an exception in such a case.  
The significance of CRS in GeoJSON has been diminished anyway, since the [new GeoJSON standard](https://tools.ietf.org/html/rfc7946) published by IETF removed the CRS support from GeoJSON:

> Specification of coordinate reference systems has been removed,
>       i.e., the "crs" member of [GJ2008] is no longer used.

We can still keep the CRS support though.